repo/packages/S/sysdig-cloud/1/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "agent": "sysdig/agent:latest"
```

Comment:
I personally don't like this practice, especially as this might jump across incompatible major version.
 I saw the discussion and understand your rationale, so my proposal would be:
Maybe you could use a major/minor version?
e.g., sysdig/agent:10 . That you could update as long as it stays compatible... What do you think?